content/showcase/_index.html

```diff
@@ -20,6 +20,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/nafisazizir/uquizzle"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/ideas.png"
+                        title="Ideas"
+                        caption="<p>Open-Source screenplay writing tool that let's you rearrange scenes and create storyboard images using AI.</p><p><b>2024 Hackathon Best Interdisciplinary</b></p>"
```

Comment:
Typically, "Open-Source" is not a proper noun, so should be capitalised as "Open-source" when starting a sentence.

Reply:
Fixed :)